src/__tests__/__snapshots__/basic.test.ts.snap

```diff
@@ -3,7 +3,7 @@
 exports[`addUDFHelper basic declaration 1`] = `"function _declaration() { console.log(\\"declaration\\"); }"`;
 
 exports[`addUDFHelper basic dependencies 1`] = `
-"function _dependencies() { cild1() || _child2(); }
+"function _dependencies() { _child() || _child2(); }
```

Comment:
It didn't work for child1

Cause unknown

Since it is a problem that you need to devise the name of the helper, hold it once